src/pages/NewChatConfirmPage.tsx

```diff
@@ -0,0 +1,152 @@
+                            />
+                        );
+                    }
+
+                    const section: ListItem = {
+                        login: selectedOption?.login ?? '',
+                        text: selectedOption?.text ?? '',
+                        keyForList: selectedOption?.keyForList ?? '',
+                        isSelected: !isAdmin,
+                        isDisabled: isAdmin,
+                        rightElement: roleBadge,
+                        accountID,
+                        icons: selectedOption?.icons,
+                    };
+                    return section;
+                })
+                .sort((a, b) => a.text?.toLowerCase().localeCompare(b.text?.toLowerCase() ?? '') ?? -1),
+        [selectedOptions, personalData.accountID, translate, styles.textStrong, styles.justifyContentCenter, styles.badgeBordered, styles.activeItemBadge, StyleUtils],
+    );
+
+    /**
+     * Removes a selected option from list if already selected.
+     */
+    const unselectOption = (option: ListItem) => {
+        if (!newGroupDraft) {
+            return;
+        }
+        const newSelectedParticipants = newGroupDraft.participants.filter((participant) => participant.login !== option.login);
+        Report.setGroupDraft(newSelectedParticipants);
+    };
+
+    const createGroup = () => {
+        if (!newGroupDraft) {
+            return;
+        }
+        const logins: string[] = newGroupDraft.participants.map((participant) => participant.login);
+        Report.navigateToAndOpenReport(logins, true, groupName);
+    };
+
+    const navigateBack = () => {
+        Navigation.goBack(ROUTES.NEW_CHAT);
+    };
+
+    return (
+        <ScreenWrapper testID={NewChatConfirmPage.displayName}>
+            <HeaderWithBackButton
+                title={translate('common.group')}
+                onBackButtonPress={navigateBack}
+            />
+            <View style={styles.avatarSectionWrapper}>
+                <Avatar
+                    containerStyles={[styles.avatarXLarge, styles.mb3]}
+                    imageStyles={[styles.avatarXLarge]}
+                    source={ReportUtils.getDefaultGroupAvatar()}
+                    size={CONST.AVATAR_SIZE.XLARGE}
+                />
+            </View>
+            <MenuItemWithTopDescription
+                title={groupName}
+                interactive={false}
+                shouldCheckActionAllowedOnPress={false}
+                description={translate('groupConfirmPage.groupName')}
+            />
+            <SelectionList
+                sections={[{data: sections, indexOffset: 0}]}
+                ListItem={TableListItem}
+                onSelectRow={unselectOption}
+                showConfirmButton={selectedOptions.length > 1}
+                confirmButtonText={translate('newChatPage.startGroup')}
+                onConfirm={createGroup}
```

Comment:
We missed adding a condition here to check if anything is selected before calling `onConfirm` - this caused an [issue](https://github.com/Expensify/App/issues/39323) where CMD+enter can still proceed past the step.